### DISCUSSION TITLE
I can not search the docs plugin and filter operators site in doc plugin is broken, why?

### DISCUSSION THREAD

**Discussion by Parlazo** [TiddlyWiki/TiddlyWiki5#5676] [Q&A]
Hi

i did download a new TiddlyWiki
Install the TW5 documentation plugin, highlight.js, raildoad, KaTex
now i type widget into the sidebar search and nothing is found, i type filter and nothing is found
in the documentation installed by the plugin i go to reference and to filters and open filter operators, the site appears but the table is empty where all operators should appear

i use firefox on windows

odd.. i search about and nothing appears, i open bout and edit/save it and now the search can find it, the only difference i see is the name of the last edit autor has changed ,,, but i dont want to open and save every file to get it into the search

so ... how can i fix this?
is the search broken?
is there a way to mass open and save the doc tiddlers?
is my tiddlerwiki broken?
is it because i did update firefox?
is it because a other plugin is disturbing the documentation plugin?

i did delete cookie and lsa storge and gownload tiddlerwiki many times and reinstall it and did use different names

any idea how to fix this?

the online search seems to work but not offline in a new wiki with the plugin

**Comment by pmario** [comment]
IMO the problem is the default search doesn't search in shadow tiddlers.  I think, this should be an issue and not a discussion

**Reply by Parlazo** [reply]
Ahh ... thanks a very lot
Yes this is an issue ... i will file it later
i was not sure if it is just me because short ago i was able to search the shadow tiddlers of the doc plugin and since shortly i noticed that some thing is broken, so i did suspect its my fault because i may have done some thing wrong.
so i was searching google an every thing and no one said some thing about this issue and thats how i got to here.

so until i can file the issue and until it gets fixed i have a may more general question on this:
is it possible to modify the default search so that it can search for shadow tiddlers but only inside of the docs plugin or may any custom path, like $:/plugin or myFileSection/thing/ ?
this would be very helpful too.

Thanks any way for telling me its not my fault, i really got crazy about this removed all tags and toc and even tenamed my tiddlers in all kind of ways because i had the idea it is my fault some how... set up a new tiddlerWiki and moved all content and every thing ... ahhhrrr

**Comment by Parlazo** [comment]
OK i did file the issue as suggested at https://github.com/Jermolene/TiddlyWiki5/issues/5678#issue-884402079
I hope this will help now some how
Thanks
